manim/scene/scene_file_writer.py

```diff
@@ -399,7 +396,7 @@ def is_already_cached(self, hash_invocation):
             return False
         path = os.path.join(
             self.partial_movie_directory,
-            "{}{}".format(hash_invocation, config["movie_file_extension"]),
+            f"{hash_invocation}{config['movie_file_extension']}",
```

Comment:
Is there a reason for changing " into ' for movie_file_extension?

Reply:
No particular reason. The string was delimited by double quotes, so I used the single quotes for the inner part.

Reply:
okay!